### ISSUE TITLE
Documentation - Bulk Import Emoji Scheme Wrong

### ISSUE THREAD

**Issue by 7-plus-t** [mattermost/mattermost-server#10395]
#### Summary
Documentation for emoji bulk import scheme is flawed and causes import to hang.

#### Steps to reproduce
Go to documentation at: https://docs.mattermost.com/deployment/bulk-loading.html#emoji-object

Scheme described by documentation is as follows:
```
{
  "name": "custom-emoji-troll",
  "image": "bulkdata/emoji/trollolol.png"
}
```

Using this scheme with the bulk import, the importer gets "stuck" and gets stuck importing the first entry.

#### Expected behavior
- Documentation matches actual implementation.
- Bulk import exits with error when invalid jsonl format is encountered

#### Observed behavior (that appears unintentional) 
- Documented scheme for emojis is wrong
- Bulk import gets stuck when documented scheme is used

#### Possible fixes
Fix documentation with correct scheme:
```
{ 
  "type": "emoji", 
  "emoji": {
    "name": "custom-emoji-troll",
    "image": "bulkdata/emoji/trollolol.png"
  }
}
```

**Comment by 7-plus-t** [comment]
Moved documentation related part of this issue to: https://github.com/mattermost/docs/issues/2579
This leaves the server part of this issue, the importer getting stuck when importing lines missing the "type" entry. I've already written a unit test reproducing the issue and will attempt to fix the importer. It is actually the case for any line missing the type, not only emojis.

**Comment by amyblais** [comment]
Hi @7-plus-t, you mentioned that you were attempting to find a fix for this issue that you found. Have you had luck so far with that and let me know if we can help with anything?

**Comment by 7-plus-t** [comment]
Hi @amyblais, I've written a unit test that fails. I've also written a fix that makes the unit test pass, but is not ready for a PR yet:
I'm not sure if the bulk loading should simply skip the lines without type attribute or fail with a parsing error. What do you think?

**Comment by amyblais** [comment]
@grundleborg do you know the answer to the above question ^ ?

**Comment by grundleborg** [comment]
If the line is missing the type it should fail with a parsing error.

**Comment by amyblais** [comment]
Ticket for tracking: https://mattermost.atlassian.net/browse/MM-16548.

**Comment by amyblais** [comment]
Closing as this has been fixed for upcoming v5.14 release,
